components/core/core-base/src/main/java/org/eclipse/dirigible/components/base/http/access/DirigibleRole.java

```diff
@@ -0,0 +1,16 @@
+package org.eclipse.dirigible.components.base.http.access;
+
+public enum DirigibleRole {
+
+    DEVELOPER("DEVELOPER"), OPERATOR("OPERATOR");
```

Comment:
Shouldn't the values be `Developer` and `Operator`?

Reply:
this what is defined in https://github.com/eclipse/dirigible/blob/master/components/security/security-basic/src/main/java/org/eclipse/dirigible/components/security/basic/BasicAuthSecurityConfiguration.java#L72 that's why I used it